feature-libs/cart/import-export/core/model/import-export.model.ts

```diff
@@ -0,0 +1,13 @@
+import { CmsComponent } from '@spartacus/core';
+
+export enum CartTypes {
+  NEW_SAVED_CART,
+  EXISTING_SAVED_CART,
+  ACTIVE_CART,
```

Comment:
1. Let's assign values to those enum keys, for better debuggability (and to be insensitive to changing order of keys).
2. What about simply `SAVED_CART` instead of `EXISTING_SAVED_CART`?